src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.scss

```diff
@@ -12,6 +12,7 @@
 
     &.bcs-is-pending {
         opacity: .4;
+        pointer-events: none;
```

Comment:
Should we set `cursor: default` as well?

Reply:
I don't think we need to, we should not have any interactivity while the pending card is showing.